velox/docs/monthly-updates/2024/december-2024.rst

```diff
@@ -0,0 +1,84 @@
+********************
+December 2024 Update
+********************
+
+This update was generated with the assistance of AI. While we strive for accuracy, please note
+that AI–generated content may not always be error-free. We encourage you to verify any information
+that is important to you.
+
+Core Library
+============
+
+* Add support for spiller abstraction with better memory management. :pr:`11656`
+* Add priority-based memory reclaim framework for improved resource management. :pr:`11598`
+* Add support for auto scale writer with load balancing. :pr:`11702`
+* Add support for key/value operations in radix sort for breeze. :pr:`11733`
+* Add support for string type key in prefix sort operations. :pr:`11527`
+* Add T-Digest data structure for advanced statistical computations. :pr:`11665`
+* Add support for UnsafeRow and CompactRow compression. :pr:`11497`
+* Add utilities for combining dictionary wrappers. :pr:`11944`
+* Add support for viewfs file system in velox. :pr:`11811`
+* Fix flaky HashJoinTest.reclaimDuringOutputProcessing test. :pr:`11993`
+* Fix writer reclaim failure by clearing dict chained buffers during arbitration. :pr:`11988`
+* Fix BitSet size to be larger than int32 for large datasets. :pr:`11986`
+* Fix simd build for different platform compatibility. :pr:`11971`
+* Fix memory manager test instance initialization. :pr:`11926`
+* Fix task hanging under serialized execution mode. :pr:`11747`
+* Fix zero reclaimable bytes handling in priority reclaiming. :pr:`11925`
+
+Presto Functions
+================
+
+* Add :func:`ip_prefix` function with cast operators for IP address operations. :pr:`11514`, :pr:`11481`, :pr:`11546`
```

Comment:
Should this something like: Add IP Address type with CAST support and ip_prefix function?